Sources/Vehicle/Templates/CarPlayPaginatedListTemplate.swift

```diff
@@ -11,13 +11,14 @@ final class CarPlayPaginatedListTemplate {
     private var items: [CPListItem]
     private var currentPage: Int
     private let title: String
+    private let isInsideTabBar: Bool
 
-    private let itemsPerPage: Int = CPListTemplate.maximumItemCount
     private(set) var template: CPListTemplate
 
-    init(title: String, items: [CPListItem]) {
+    init(title: String, items: [CPListItem], isInsideTabBar: Bool = false) {
```

Comment:
this parameter should describe what it does, not how it is. e.g. make it an `enum AdvancementType { case navigationItems, inlineItems }` and provide that